test-cypress/package.json

```diff
@@ -4,7 +4,8 @@
     "description": "Cypress tests for GraphDB workbench",
     "scripts": {
         "start": "cypress open",
-        "test": "cypress run"
+        "test": "cypress run",
+        "test:partial": "cypress run --spec \"integration/repository/**\""
```

Comment:
Do you intend to leave this? I like it and if you parametrize it it will be great :)

Reply:
Not with intention, but since you like it I will leave it :)